packages/ember/addon/index.ts

```diff
@@ -17,6 +17,9 @@ export function InitSentryForEmber(_runtimeConfig: BrowserOptions | undefined) {
   assert('Missing configuration for Sentry.', config.sentry);
 
   const initConfig = Object.assign({}, config.sentry, _runtimeConfig || {});
+  if (initConfig.autoSessionTracking === undefined) {
```

Comment:
This is unnecessary, as we call `Sentry.init` which already does that 3 lines below.